src/themes/index.d.ts

```diff
@@ -46,6 +52,7 @@ declare module "@mui/material/styles" {
   interface BreakpointOverrides extends FederalBreakpointOverrides {}
 
   interface Palette {
+    cobalt: FederalColor & { main: string };
```

Comment:
Question: cobalt ? Is this intended ? Asking since the commit this change is included in is about tertiary.